### DISCUSSION TITLE
How to use django-types with VSCode (pylance)?

### DISCUSSION THREAD

**Discussion by ahmedbilal** [sbdchd/django-types#148] [Q&A]
How to use django-types with VSCode (pylance)? If I install django-types as dependency, VSCode doesn't seems to use that probably because the pylance is shipped with django-stubs.

**Comment by sbdchd** [comment]
Honestly not sure, I've been having the same issue

related: https://github.com/microsoft/pylance-release/issues/2559

I think it's really in pylance's court to fix since we have a `py.typed` file in this package and it's a stub package
